jsonschema/tests/test_validators.py

```diff
@@ -1341,6 +1337,16 @@ def fake_urlopen(url):
             pass
         self.assertEqual(resolved, 12)
 
+    def test_it_retrieves_local_refs_via_urlopen(self):
+        fd, tempf = tempfile.mkstemp()
```

Comment:
`tempfile.NamedTemporaryFile(delete=False)` might knock a line or two off of this.

Reply:
right, done